hack/release.go

```diff
@@ -76,12 +95,12 @@ func getEnviromnentVariableOrDefault(defaultValue, varName string) string {
 	return defaultValue
 }
 
-func initWithEnvVariale(name string, image *mellanoxv1alpha1.ImageSpec) {
-	envName := name + "_IMAGE"
+func initWithEnvVariale(param string, image *ReleaseImageSpec) {
```

Comment:
why the rename to "param" ? is it clearer this way ?

Reply:
It was shadowing an import. Renamed the import and reverted this.